src/cf-modals/usage.md

```diff
@@ -0,0 +1,175 @@
+                </button>
+                <h1 id="example-modal-title-1">Lorem ipsum</h1>
+                <div id="example-modal-1-desc">
+                    <p>
+                        Sed ut perspiciatis unde omnis iste natus error sit voluptatem accusantium doloremque laudantium, totam rem aperiam, eaque ipsa quae ab illo inventore veritatis et quasi architecto beatae vitae dicta sunt explicabo. Nemo enim ipsam voluptatem quia voluptas sit aspernatur aut odit aut fugit, sed quia consequuntur magni dolores eos qui ratione voluptatem sequi nesciunt.
+                    </p>
+                    <p>
+                        Quis nostrum exercitationem ullam corporis suscipit laboriosam,
+                        nisi ut aliquid ex ea commodi consequatur?
+                    </p>
+                </div>
+            </div>
+        </div>
+    </div>
+</div>
+
+```
+<div class="o-modal"
+     aria-hidden="true"
+     role="dialog"
+     aria-labelledby="example-modal-1-title"
+     aria-describedby="example-modal-1-desc">
+    <div class="o-modal_backdrop"></div>
+    <div class="o-modal_container">
+        <div class="o-modal_content">
+            <div class="o-modal_body">
+                <button class="o-modal_close a-btn a-btn__link">
+                    Close
+                    <span class="cf-icon cf-icon-delete-round"></span>
+                </button>
+                <h1 id="example-modal-title-1">Lorem ipsum</h1>
+                <div id="example-modal-1-desc">
+                    <p>
+                        Sed ut perspiciatis unde omnis iste natus error sit voluptatem accusantium doloremque laudantium, totam rem aperiam, eaque ipsa quae ab illo inventore veritatis et quasi architecto beatae vitae dicta sunt explicabo. Nemo enim ipsam voluptatem quia voluptas sit aspernatur aut odit aut fugit, sed quia consequuntur magni dolores eos qui ratione voluptatem sequi nesciunt.
+                    </p>
+                    <p>
+                        Quis nostrum exercitationem ullam corporis suscipit laboriosam,
+                        nisi ut aliquid ex ea commodi consequatur?
+                    </p>
+                </div>
+            </div>
+        </div>
+    </div>
+</div>
+```
+
+### Alert modal
+
+This version of a modal uses a form for the `o-modal_content` and includes
+a footer that allows the user to perform an action, such as saving
+entered content. Unlike a dialog model, the first content a user should
+interact with is focused, which may be the first occurance of an `<input>`,
+`<submit>`, `<button>` or the close button designated by
+the `o-modal_close` class.
+
+<div class="o-modal"
+     aria-hidden="true"
+     role="alertdialog"
+     aria-labelledby="example-modal-2-title"
+     aria-describedby="example-modal-2-desc">
+    <div class="o-modal_backdrop"></div>
+    <div class="o-modal_container">
+        <form class="o-modal_content">
```

Comment:
should this be `o-modal_form`?

Reply:
I don't think it needs to be. It only needs to support a form when there are inputs and a submit button, which may not always be the case. There's no extra styling outside of `o-modal_content`.